test/interaction/Issue6261.out

```diff
@@ -0,0 +1,36 @@
+(agda2-status-action "")
+(agda2-info-action "*Type-checking*" "" nil)
+(agda2-highlight-clear)
+(agda2-info-action "*Type-checking*" "Checking Issue6261 (Issue6261.agda). " t)
+(agda2-info-action "*Type-checking*" " Checking Issue6261a (Issue6261a.agda). " t)
+(agda2-info-action "*Type-checking*" " Finished Issue6261a. " t)
+(agda2-info-action "*Type-checking*" " Checking Issue6261b (Issue6261b.agda). " t)
+(agda2-info-action "*Type-checking*" " Checking Issue6261c (Issue6261c.agda). " t)
+(agda2-info-action "*Type-checking*" " Finished Issue6261c. " t)
+(agda2-info-action "*Type-checking*" " Finished Issue6261b. " t)
+(agda2-info-action "*Type-checking*" "Finished Issue6261. " t)
+(agda2-status-action "Checked")
+(agda2-info-action "*All Done*" "" nil)
+((last . 1) . (agda2-goals-action '()))
+(agda2-status-action "")
+(agda2-info-action "*Type-checking*" "" nil)
+(agda2-highlight-clear)
+(agda2-info-action "*Type-checking*" "Loading Issue6261 (_build/2.6.4/agda/Issue6261.agdai). " t)
+(agda2-info-action "*Type-checking*" " Loading Issue6261a (_build/2.6.4/agda/Issue6261a.agdai). " t)
```

Comment:
This message apparently can come later, there is a race condition on CI:
https://github.com/agda/agda/actions/runs/5765746926/job/15632876653#step:11:201 reports the following diff:
```
 (agda2-info-action "*Type-checking*" "Loading Issue6261 (_build/2.6.4/agda/Issue6261.agdai). " t)
-(agda2-info-action "*Type-checking*" " Loading Issue6261a (_build/2.6.4/agda/Issue6261a.agdai). " t)
 (agda2-info-action "*Type-checking*" " Loading Issue6261b (_build/2.6.4/agda/Issue6261b.agdai). " t)
 (agda2-info-action "*Type-checking*" " Loading Issue6261c (_build/2.6.4/agda/Issue6261c.agdai). " t)
+(agda2-info-action "*Type-checking*" " Loading Issue6261a (_build/2.6.4/agda/Issue6261a.agdai). " t)
```